app.arc

```diff
@@ -72,6 +72,16 @@ auto_increment_metadata
   tableName *String
   PointInTimeRecovery true
 
+circulars_history
+  circularId *Number
+  version **Number
+  PointInTimeRecovery true
+
+circulars_change_requests
+  circularId *Number
+  requestorSub **String
+  PointInTimeRecovery true
```

Comment:
Will this data model limit us to one change request per circular per user at a time?

Reply:
Yes. I think that makes the most sense for this use case. Once approved, this just sets the subject and body to whatever the request has in it, so if they make multiple requests at the same they will result in overwriting. 

If we see more of a need for some granularity in the future we could implement more of a diff based approach